lib/RichtextRenderer.astro

```diff
@@ -0,0 +1,38 @@
+---
+import { renderRichText } from "./";
+import type { ISbRichtext, SbBlokData, SbRichTextOptions } from "./dist/types";
+
+import StoryblokComponent from "./StoryblokComponent.astro";
+
+export interface Props {
+  richtextObject: ISbRichtext;
```

Comment:
@Baheya @manuelschroederdev what do you think on calling it `richTextData` and `richTextOptions? Reasons:
- `Object` is too tied to a specific shape of the data
- We already use camelCase for `renderRichText`, which was agreed in the past

Reply:
Completely agree 🙂

Reply:
@alexjoverm those are very good points! I will update accordingly :)